build/buildspec.docker.master.yml

```diff
@@ -35,6 +35,8 @@ phases:
       - docker pull minio/minio:latest
       - docker pull mcr.microsoft.com/azure-storage/azurite:latest
       - docker pull fsouza/fake-gcs-server:latest
+      - docker pull atzoum/simulator-google-sheets:latest
```

Comment:
@lvrach can we fork this repo to prepare a rudderstack image?

Reply:
Sure we can, but we can also push the changes upstream. I believe the original author will be happy to accept them.